wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -1095,7 +1096,7 @@ class KotlinGeneratorTest {
     val code = repoBuilder.generateKotlin(longType)
     assertTrue(code.contains("return false"))
     assertTrue(code.contains("return $longType("))
-    assertTrue(code.contains("$longMember =="))
+    assertTrue(code.contains("\\($longMember\\s+!=\\s+other.$longMember\\)".toRegex()))
```

Comment:
Why are you using a regex? Tough to read.

Reply:
The long expression is on multiple lines. I wanted to check the same part of the code as before. Copy&paste generated code seems to brittle to me

Reply:
Even multiple line, if it breaks, it will be easier to use if a plain string instead of a regex.

Reply:
Fixed, I am not sure about the indenation